assets/tests/components/routePicker.test.tsx

```diff
@@ -18,6 +18,11 @@ describe("RoutePicker", () => {
         directionNames: { 0: "Outbound", 1: "Inbound" },
         name: "111",
       },
+      {
+        id: "741",
+        directionNames: { 0: "Outbound", 1: "Inbound" },
+        name: "SL1",
```

Comment:
If you added this to test the name being used in the selected list, you'll also need to add it to the `selectedRouteIds` below.